doc/authoring_command_modules/README.md

```diff
@@ -225,12 +225,12 @@ An example title of non-customer-facing change PR:
 
 ### Format PR Description
 
-If you would like to write multiple history notes for one PR, please write the notes under `History Notes` section in the PR description,  following the same format described above. In this case, the PR title should be a summary of all the changes in this PR and will not be put into `HISTORY.rst`.
+If you would like to write multiple history notes for one PR, please write the notes under `History Notes` section in the PR description, following the same format described above. The PR template already contains the history note template, just change it if needed. In this case, the PR title should be a summary of all the changes in this PR and will not be put into `HISTORY.rst`. You can delete the `History Notes` section if not needed.
```

Comment:
Perhaps it will be good to describe where to put issue ID or the `Fix #xxx` phrase?

Reply:
Added in the same part as BREAKING CHANGE.